### PR TITLE
RNP-60 Bump the iOS SDK version in the RN plugin

### PR DESCRIPTION
**Pull request by kfa-onewelcome**

### CODE REVIEW

File: ios/RNOneginiSdk.swift [hunk 1]
```diff
@@ -177,9 +177,10 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
     func authenticateUser(_ profileId: (NSString),
                         resolver resolve: @escaping RCTPromiseResolveBlock,
                         rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
-        let profile: ONGUserProfile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!;
-
-        bridgeConnector.toLoginHandler.authenticateUser(profile) {
+        let profile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!
```
Comment (szymon-onewelcome) [inline ios/RNOneginiSdk.swift:180]: Are we sure about this force unwrapping (here and in fallowing lines)?

File: ios/RNOneginiSdk.swift [hunk 2]
```diff
@@ -221,7 +222,7 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
                                     scopes: [String],
                                     resolver resolve: @escaping RCTPromiseResolveBlock,
                                     rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
-        let profile: ONGUserProfile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!;
+        let profile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!;
```
Comment (szymon-onewelcome) [inline ios/RNOneginiSdk.swift:225]: Can you define an object property (`let userClient = ONGClient.sharedInstance().userClient`) and use it everywhere? Or even function that would return `profile` for provided `profileId`?

**Review by szymon-onewelcome — CHANGES_REQUESTED** [review]

File: ios/RNOneginiSdk.swift (hunk 1 above)
Comment (kfa-onewelcome) [reply ios/RNOneginiSdk.swift:180]: I'm sure because this is not my code :)

File: ios/RNOneginiSdk.swift (hunk 2 above)
Comment (kfa-onewelcome) [reply ios/RNOneginiSdk.swift:225]: I could but this is not a scope of the story.

Comment (kfa-onewelcome) [reply ios/RNOneginiSdk.swift]: I've refactored nevertheless 🤟

File: ios/Handlers/RegistrationHandler.swift [hunk 3]
```diff
@@ -138,12 +131,20 @@ extension RegistrationHandler: ONGRegistrationDelegate {
 
         sendCustomRegistrationNotification(CustomRegistrationNotification.finishRegistration, result)
     }
+    
+    func userClient(_ userClient: ONGUserClient, didRegisterUser userProfile: ONGUserProfile, identityProvider: ONGIdentityProvider, info: ONGCustomInfo?) {
+        createPinChallenge = nil
+        customRegistrationChallenge = nil
+        signUpCompletion!(true, userProfile, nil)
```
[the remark this answers is not in the document]
Comment (kfa-onewelcome) [reply ios/Handlers/RegistrationHandler.swift:138]: I can create a story for the refactor, but these changes are not for now (I've already applied some of your remarks)

**Review by szymon-onewelcome — APPROVED** [review]

**Pull request merged**